src/Core/Usecase/Notification/CreateNotification.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Add notification notification
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Usecase\Notification;
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Usecase\CreateUsecase;
+
+class CreateNotification extends CreateUsecase
```

Comment:
What differs here from the parent CreateUsecase? - can you call that out in a comment?